prime-router/src/main/kotlin/fhirengine/engine/FHIRTranslator.kt

```diff
@@ -114,6 +121,11 @@ class FHIRTranslator(
         logger.trace("Preparing to send original message")
         val originalReport = reportService.getRootReport(message.reportId)
         val bodyBytes = BlobAccess.downloadBlobAsByteArray(originalReport.bodyUrl)
```

Comment:
nit: Have we considered updating this to use `BlobAccess.downloadBlob(...)` which will do the digest check for us?

Reply:
Made the change.